### ISSUE TITLE
Update the docs for kafkasource usage

### ISSUE THREAD

**Issue by Shashankft9** [knative/docs#2520]
## Expected Behavior
The logs in the Aache Kafka Event Source (4th step) and Verify (step 2) sections are not updated after the new knative release: https://github.com/knative/docs/tree/master/docs/eventing/samples/kafka/source

4th step log should be from here: 
https://github.com/knative/eventing-contrib/blob/1db4476e8a13bad85e19d9afc7e443b80e75170e/kafka/source/pkg/adapter/adapter.go#L81

while the 2nd step log should be from here:   
https://github.com/knative/eventing-contrib/blob/master/kafka/common/pkg/kafka/consumer_handler.go#L87

## Actual Behavior
Its showing logs from previous releases, I cant pin down of which past release, since I have started using knative from 0.14.0

## Additional Info
**Additional context**
I have already forked the branch and have done the changes so as to submit a PR following the conversation on slack, but there is a blocker discussed here: https://knative.slack.com/archives/C93E33SN8/p1590672378060600?thread_ts=1590652198.043900&cid=C93E33SN8
Basically the verify logs are level debug, so in order to see them the user must edit the configmap `config-logging` in `knative-sources` ns and then go on to manually delete the deployments of kafkasources, then only the logs will show up. The question is, where to include these additional steps - here only or a separate document?  

**Install information:**
 - Platform (GKE, IKS, AKS, etc.): kubernetes 
 - Knative Version: v0.14.0
